components/http_kernel.rst

```diff
@@ -124,9 +124,9 @@ For general information on adding listeners to the events below, see
 
 .. caution::
 
-    As of 3.1 the :class:`Symfony\\Component\\Httpkernel\\HttpKernel` accepts a
+    As of 3.1 the :class:`Symfony\\Component\\HttpKernel` accepts a
```

Comment:
I'd say that the original reference was correct. See: http://api.symfony.com/3.1/Symfony/Component/HttpKernel/HttpKernel.html